src/applications/domains-and-routing.md

```diff
@@ -20,3 +20,13 @@ appname to the old one:
     ("start-of-authority") records. It is highly recommended that you bind a subdomain to
     an application, however you can work around this by pointing the @ record to the
     address of the load balancer (if any).
+
+To add or remove your application from the routing mesh, use `deis routing`:
+
+    $ deis routing:disable
+    Disabling routing for finest-woodshed... done
+
+This will make your application unreachable through the [Router][].
```

Comment:
Since we reference "add or remove" it might make sense to include the enable example as well.

Should also note that when routing is "disabled" the application is still reachable via its Kubernetes service.